var/spack/repos/builtin/packages/octopus/package.py

```diff
@@ -114,11 +165,54 @@ def install(self, spec, prefix):
                 '--with-scalapack=%s' % spec['scalapack'].libs
             ])
 
+        if '+cgal' in spec:
+            args.extend([
+                '--with-cgal-prefix=%s' % spec['cgal'].prefix,
+            ])
```

Comment:
FWIW.  You could use `append` when only one option/string is being added to the args:

```suggestion
            args.append(
                '--with-cgal-prefix=%s' % spec['cgal'].prefix
            )
```

Reply:
Good point - let's keep it as simple as possible. That's implemented in https://github.com/spack/spack/pull/30321/commits/b9035495716dbb0abb672a478b645e1c970b4c9b .